SQL/database_changelog.txt

```diff
@@ -8,6 +8,14 @@ INSERT INTO `SS13_schema_revision` (`major`, `minor`) VALUES (6, 1);
 
 In any query remember to add a prefix to the table names if you use one.
 
+-----------------------------------------------------
+
+Version 6.1-SOUNDTRACK - 11 March 2023 by itsmeow - Minor update, enable soundtrack pref for existing users
+
+UPDATE SS13_preferences SET preference_value = preference_value | (1<<13) WHERE preference_tag = 2;
+
+-----------------------------------------------------
```

Comment:
Can't we just use the version based prefs updating, or was that removed?

Reply:
That was for savefiles